cross/he853/PLIST

```diff
@@ -0,0 +1,2 @@
+bin:bin/he853
+rules.d:/lib/udev/rules.d/80-he853.rules
```

Comment:
With the edit on the line https://github.com/SynoCommunity/spksrc/pull/2672/commits/17cc84d578c383d31ed67b9dd880d2703a3ecd7a#diff-cfe0eac3473ecdc060a3305db727c762R22
You have to change this line by this : (remove the `/` at start and replace `rules.d` by `lib`)

    lib:lib/udev/rules.d/80-he853.rules